src/http/http.cpp

```diff
@@ -17,15 +17,19 @@ std::vector<std::thread> workers = {};
 
 } // namespace
 
-void tfs::http::start(std::string_view address, unsigned short port /*= 8080*/, int threads /*= 1*/)
+void tfs::http::start(bool bindOnlyOtsIP, std::string_view otsIP, unsigned short port /*= 8080*/, int threads /*= 1*/)
```

Comment:
Let's use `ConfigManager` here then, instead of growing the list of arguments. In retrospect, it should have been like that since always.

Reply:
This PR is to make TFS work without Docker and with Docker the same way.
This probably also solves problem of running public server on Windows with IP behind DMZ/port forward.

We can change `ConfigManager` in the next PR, but this should change the code in all places that use/not use `ConfigManager`, not just on the HTTP server. To make it consistent across TFS code.
It may be a part of https://github.com/otland/forgottenserver/issues/4963 solution.